client/lib/url/index.js

```diff
@@ -20,4 +21,4 @@ function isExternal( url ) {
 	return isOutsideCalypso( url ) && ! startsWith( url, '//wordpress.com' );
 }
 
-export default { isOutsideCalypso, isExternal };
+export default { isOutsideCalypso, isExternal, support };
```

Comment:
Do we really need to export `support` here? You import `lib/url/support` directly in other files anyway.